server/services/participants.js

```diff
@@ -141,6 +141,16 @@ const getParticipants = async (user, pagination, sortField,
     };
   }
 
+  const callbackStatus = (participant) => {
+    const d = new Date(participant.updated_at);
+    if (d instanceof Date && !Number.isNaN(d.getTime())) {
```

Comment:
Doesn’t will `d` be always an instance of Date?

Reply:
It should be, yes.